wazo_agid/handlers/outgoing_callerid_formatter.py

```diff
@@ -17,14 +18,18 @@
 logger = logging.getLogger(__name__)
 
 
-def _remove_none_numeric_char(raw: str) -> str:
-    return ''.join(c for c in raw if c.isdigit())
+def _remove_non_numeric_char(raw: str | None) -> str | None:
+    return ''.join(c for c in raw if c.isdigit()) if raw else None
```

Comment:
don't accept `None` values if just passing them through(no effect on the logic). The function's caller should call this function only if the value is non-null.